community/STEERING-COMMITTEE.md

```diff
@@ -1,64 +1,156 @@
+and overall project strategy. Because of the nature of the project and funding required,
+it was decided that strong corporate leadership was necessary for the project to ensure
+velocity.
+
+_What happens if someone leaves the company they represent?_
 
-<!-- TODO ## Committee Meeting -->
+In that case, the company will nominate a new representative for the seat. The nominee
+shall be confirmed by the standard process as called out above.
 
-## Committee Members
+_What happens if someone leaves the project or decides to leave the committee?_
 
-The members of the Steering Committee are shown below. Membership in the SC is
-determined by current level of contribution to the project. Contribution is
-periodically reviewed to ensure proper recognition.
+As above, the same would apply. The company who holds the seat will nominate a new
+representative to hold the seat. The nominee shall be confirmed by the standard process,
+called out above.
 
-| &nbsp;                                                   | Member         | Company | Profile                                  |
-| -------------------------------------------------------- | -------------- | ------- | ---------------------------------------- |
-| <img width="30px" src="https://github.com/dewitt.png">   | DeWitt Clinton | Google  | [@dewitt](https://github.com/dewitt)     |
-| <img width="30px" src="https://github.com/mchmarny.png"> | Mark Chmarny   | Google  | [@mchmarny](https://github.com/mchmarny) |
-| <img width="30px" src="https://github.com/isdal.png">    | Tomas Isdal    | Google  | [@isdal](https://github.com/isdal)       |
+_Is this steering model permanent?_
+
+The governance of Knative is an open, living document, and will continue to evolve as
+the community and project change. We expect over time we will adapt the way we run this
+committee, based on feedback from the community.
+
+_How are steering committee decisions reached?_
+
+The steering committee desires to always reach consensus. Decisions are made in
+meetings when a quorum of the members are present and may pass with majority
+of the committee supporting it. Quorum is considered reached when at least half
+of the members are present.
+
+The committee's charter may be changed by majority consent of committee members.
+
+In case of extended absence, an absent member will be considered to be in
+agreement after eight days have passed since a proposal was accepted by all
+present members. The company of the absent member may appoint a single delegate
+from the same company, during absences.
+
+## Getting in touch
+
+If you'd like to reach out to the committee, please drop a note
+to [knative-steering@googlegroups.com](mailto:knative-steering@googlegroups.com).
+
+All members of the KSC have access to the messages sent to this address.
 
 ---
 
+Portions of this document are adapted from the
+[Istio Steering Committee](https://github.com/istio/community/blob/master/STEERING-COMMITTEE.md)
```

Comment:
The Kubernetes community took a long time to develop their SC and the process to do its election.  I would be in favor of a uniform set of governance rules.

Let's just adopt their mechanisms and be consistent within the overall Kubernetes community

Reply:
I don't disagree as a starting point, but I would want to make clear that we can amend the rules independently of Kubernetes and that rule amendments by Kubernetes don't necessarily flow to Knative rules. Put another way: a hard fork, not a tracking branch.

Reply:
I have nothing against Istio, but I believe that the k8s process (which has not been easy) has shown success. It should not be "Istio is using this so let's use the same", when this is still unproven.